test/e2e/environment/project_test.go

```diff
@@ -58,6 +60,41 @@ func TestListProjects(t *testing.T) {
 	}
 }
 
+func TestListProjectsRequestOrganizations(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+	c := newEnvironmentClient(t)
+	defer c.Close()
+	pageSize := int64(1)
+
+	orgResp, err := c.ListOrganizations(ctx, &environmentproto.ListOrganizationsRequest{PageSize: pageSize})
+	if err != nil {
+		t.Fatal(err)
+	}
+	length := int(math.Min(float64(len(orgResp.Organizations)), 2))
+	orgIds := make([]string, length)
+	for i := 0; i < length; i++ {
+		orgIds[i] = orgResp.Organizations[rand.Intn(len(orgResp.Organizations))].Id
+	}
+
+	resp, err := c.ListProjects(ctx, &environmentproto.ListProjectsRequest{PageSize: pageSize, OrganizationIds: orgIds})
+	if err != nil {
+		t.Fatal(err)
+	}
+	for _, project := range resp.Projects {
+		found := false
+		for _, orgId := range orgIds {
+			if orgId == project.OrganizationId {
+				found = true
+				break
+			}
+		}
+		if !found {
+			t.Fatalf("received a project of a not request organization expected: %v, actual: %v", orgIds, project.OrganizationId)
```

Comment:
`%s` for string.

```suggestion
			t.Fatalf("received a project of a not request organization expected: %v, actual: %s", orgIds, project.OrganizationId)
```

Reply:
Changed a specifier for variable strings.
https://github.com/bucketeer-io/bucketeer/pull/810/commits/bcb4b3ac12500790f6e344732688019cbb494090